Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1

```diff
@@ -193,7 +193,7 @@ function Delete-NonAadIdentityProvidersInApiManagementServices
     $nonApplicableSkus = @("Consumption")
 
     Write-Host $([Constants]::DoubleDashLine)
-    Write-Host "[Step 1 of 4] Prepare to delete all non-AAD identity providers (and basic authentication) from the API Management services in Subscription: [$($SubscriptionId)]"
+    Write-Host "[Step 1 of 4] Prepare to delete all non-AAD(Entra Id) identity providers (and basic authentication) from the API Management services in Subscription: [$($SubscriptionId)]"
```

Comment:
brackets need spaces before it
This gets printed out on our customer's consoles, we can't be shabby

Reply:
fixed.